### PR TITLE
Add lets take off-jhi

### PR DESCRIPTION
**Pull request by likelion-louie**
It's been a really long time to pull request

### CODE REVIEW

File: script/2020/20200313-lets-take-off-jhi.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+**스터디 회차** | **스터디 날짜** | **작성자**
+---| ---| ---|
+92회  | 2020.04.19  | 조현인 |
```
Comment (datalater) [inline script/2020/20200313-lets-take-off-jhi.md:3]: 94회로 정정해주시오~

**Comment by datalater** [comment]
코멘트 보고 수정해주시오~

**Comment by likelion-louie** [comment]
Modify wrong information

File: script/2020/20200313-lets-take-off-jhi.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+**스터디 회차** | **스터디 날짜** | **작성자**
+---| ---| ---|
+94회  | 2020.04.19  | 조현인 |
```
Comment (datalater) [inline script/2020/20200313-lets-take-off-jhi.md:3]: Great!

**Pull request merged**